tutorials/Swing_Integration/README.md

```diff
@@ -0,0 +1,152 @@
+# Integration Compose for Desktop into Swing-based aplication
+
+## What is covered
+
+In this tutorial, we will show you how to use ComposePanel in your Swing application.
+
+## Using ComposePanel
+
+ComposePanel lets you create a UI using Compose for Desktop in a Swing-based UI. To achieve that you need to create instance of ComposePanel, add it to your Swing layout and describe composition inside `setContent`. Also you may need to clear the CFD application events via `AppManager.setEvents`.
```

Comment:
ComposePanel lets you create a UI using Compose for Desktop in a Swing-based UI. To achieve this you need to create an instance of ComposePanel, add it to your Swing layout, and describe the composition inside `setContent`. You may also need to clear the CFD application events via `AppManager.setEvents`.